.gitignore

```diff
@@ -8,4 +8,5 @@ captures
 .cxx
 local.properties
 xcuserdata
-.kotlin
\ No newline at end of file
+.kotlin
+*.gpg
```

Comment:
It might be nice to add a brief comment to explicitly state why `.gpg` files are excluded?